pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py

```diff
@@ -2152,3 +2152,375 @@ def ro_dynamic_phase_seq(hard_sweep_dict, qbp_name, qbr_names,
         ps.Pulsar.get_instance().program_awgs(seq)
 
     return seq, np.arange(seq.n_acq_elements())
+
+
+## Multi-qubit time-domain sequences ##
```

Comment:
Nice job writing these sequences! 👍 
In the future, I think it would however be nice to have a generic way of expanding a sequence done on one qubit to many qubits in parallel. I guess this is close to what we can currently do on the pulse level with the blocks, but we haven't really thought of this use-case in particular.  Should we create an git issue for this feature? @stephlazar @antsr @chellings  what do you think? Because here, there is about 90% overlap with single qubit sequences, right? If for some reason we have to modify the single qubit sequences, then we'll have to modify this code too, which makes it a bit clumsy. But I think for the short term it make sense to keep it like this.

Reply:
I think we should always create general n-qubit sequences that work for the case nr_qubits = 1 :P